src/lib/components/tippies/profile_dropdown.svelte

```diff
@@ -5,7 +5,20 @@
     import User from "$icons/user.svelte";
     import { Avatar } from "@skeletonlabs/skeleton";
     import type { SvelteComponent } from "svelte";
+    // Bindings
+    let email_element_scroll_percent = 0,
+        username_element_scroll_percent = 0;
+
+    function mouseenter(el: HTMLElement) {
+        document.body.classList.add("select-none");
+        el.classList.add("select-text");
+    }
 
+    function mouseleave(el: HTMLElement) {
+        el.classList.remove("select-text");
+        document.body.classList.remove("select-none");
+    }
```

Comment:
Why? simply I dont get the point.
adding `select-*` uhh why? whats the issue?

Reply:
Try to scroll the element without using user-select. It will select all the dom nodes

Reply:
![image](https://github.com/tokitou-san/CoreProject-V3-UI/assets/61817579/e6a5dafa-9c54-4e48-aae1-1d13ea77efb8)

Reply:
![Uploading image.png…]()

Reply:
![image](https://github.com/tokitou-san/CoreProject-V3-UI/assets/61817579/933ab239-b281-43e9-925a-f89dd9fcb5cc)

Reply:
> Try to scroll the element without using user-select. It will select all the dom nodes

Okay that makes sense, but If I'm I wanna select:- just 3 clicks...?
well this also good...